main.cpp

```diff
@@ -1213,6 +1236,7 @@ void build_rmap_uf2(FILE *file, range_map<size_t>& rmap) {
             block.magic_end == UF2_MAGIC_END) {
             if (block.flags & UF2_FLAG_FAMILY_ID_PRESENT && block.file_size == RP2040_FAMILY_ID &&
                 !(block.flags & UF2_FLAG_NOT_MAIN_FLASH) && block.payload_size == PAGE_SIZE) {
+                printf("%08x\n", block.target_addr);
```

Comment:
Is this a debugging-print which you forgot to remove?

Reply:
yes